app/assets/javascripts/models/data/collection.js.coffee

```diff
@@ -125,9 +125,9 @@ ns.Collection = do (ko
       granule = @browseable_granule
       collection_id = @id for link in @links when link['rel'].indexOf('browse#') > -1
       if collection_id?
-        "#{scalerUrl}/datasets/#{collection_id}?h=85&w=85"
+        "#{scalerUrl}/datasets/#{collection_id}?h=75&w=75"
```

Comment:
Can you define the dimension in one place and use that constant? I'm uncomfortable of sprinkling the magic number 75 everywhere. Also, it seems like we've failed to update it consistently in the past.